UnitedIncome/components
# StackableExpandCollapse Window resize bugfix

### Description
> Please provide a brief description of your changes below.

The `StackableExpandCollapse` component would not render correctly (cut-off content) on certain screen sizes. This is because it was not listening for the screen resize event; this PR adds that functionality.

This hasn't been an issue because content up until now was pretty small, but the somewhat lengthy content in the related `client` PR unearthed this.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

1. Pull down this code and the related `client` PR
2. `yarn link` this code to the `client` PR, [instructions here if you are unfamiliar](https://github.com/UnitedIncome/components/blob/develop/CONTRIBUTING.md#testing-changes-)
3. Follow the testing instructions for **Scenario 1 - Testing as a Wealth Manager** from the relted `client` PR testing instructions
    - Start with a large-ish screen size (greater than 1000px) and resize past a large media breakpoint (992px). Resizing should trigger resizing and you should not see cutoff screen content in the new "Investment Plan" section.

Do not forget to `yarn unlink` when you are done!

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

Cutoff text **without** these changes
<img width="208" alt="Screen Shot 2020-05-04 at 5 30 25 PM" src="https://user-images.githubusercontent.com/8663031/81015619-006c6e00-8e2d-11ea-8b30-0c9749c286e4.png">

Proper text display **with** these changes
<img width="192" alt="Screen Shot 2020-05-04 at 5 28 12 PM" src="https://user-images.githubusercontent.com/8663031/81015641-05312200-8e2d-11ea-9928-770bada4f1c9.png">

### Comments
> Any additional comments that will aid with the review process.

I did not convert to TypeScript due to time constraints. This would be a next logical step for this component, as well as conversion to hooks and react-testing-library.

- [Related client PR](https://github.com/UnitedIncome/client/pull/3904)
